packages/gui/webpack.react.babel.ts

```diff
@@ -58,7 +60,7 @@ export default {
   context: CONTEXT,
   devtool: DEV ? 'inline-source-map' : 'source-map',
   entry: path.join(CONTEXT, '/src/index'),
-  target: 'electron-renderer',
+  target: 'web',
```

Comment:
Great!
Sorry, but this short line is what you achieved.
I just wanted to show my respect.